staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3906,9 +3908,24 @@ type ServiceSpec struct {
 	// of peer discovery.
 	// +optional
 	PublishNotReadyAddresses bool `json:"publishNotReadyAddresses,omitempty" protobuf:"varint,13,opt,name=publishNotReadyAddresses"`
+
 	// sessionAffinityConfig contains the configurations of session affinity.
 	// +optional
 	SessionAffinityConfig *SessionAffinityConfig `json:"sessionAffinityConfig,omitempty" protobuf:"bytes,14,opt,name=sessionAffinityConfig"`
+
+	// topologyKeys is a preference-order list of topology keys which clients
```

Comment:
shouldn't this be refreshed from pkg/apis/core/types.go?

Reply:
Oops, nice catch